packages/core/src/i18n.ts

```diff
@@ -173,31 +173,25 @@ export class I18n extends EventEmitter<Events> {
   }
 
   /**
-   * @param locales one locale or array of locales.
-   *   If array of locales is passed they would be used as fallback
-   *   locales for date and number formatting
+   * @param locale initial active locale.
```

Comment:
the tsdoc annotation seems wrong after the update ;) maybe just define an interface or type for the new object instead: `{
    locale: Locale
    locales?: Locales
    messages: Messages
  }`

Reply:
thanks for the catch, changed it :)